contracts/series/SeriesDeployer.sol

```diff
@@ -91,6 +123,31 @@ contract SeriesDeployer is
         addressesProvider = IAddressesProvider(_addressesProvider);
     }
 
+    /// @notice This function allows the owner address to update allowed strikes for the auto series creation feature
+    /// @param strikeUnderlyingToken underlying asset token that options are written against
+    /// @param min minimum strike allowed
```

Comment:
1 ) Can we change this to say percentage instead of allowed strike
2)  Change the vars to say minPercent/ maxPercent or something a little more detailed 
Reason is someone is coming to use this function they can determine what this is doing based on comments / params